apps/common/docker-compose.tests.yml

```diff
@@ -94,8 +94,8 @@ services:
               source: ./../postgresql-server/bin/
               target: /opt/mediacloud/bin/
             - type: bind
-              source: ./../postgresql-server/schema/
-              target: /opt/mediacloud/schema/
+              source: ./../postgresql-server/pgmigrate/
+              target: /opt/mediacloud/pgmigrate/
```

Comment:
Here and elsewhere, the mount point is wrong - given that you bake in `pgmigrate` directory to `/opt/postgresql-server/`:

https://github.com/mediacloud/backend/blob/a69189c18e723ff5b1c3c43a1bac42fd10bd4900/apps/postgresql-server/Dockerfile#L41

you should be mounting `postgresql-server/pgmigrate/` from the host to `/opt/postgresql-server/` (not `/opt/mediacloud/`) on the guest too, so that whenever `postgresql-server` service gets started from any of the `docker-compose.tests.yml`, it "sees" the latest `pgmigrate` directory from the host and not whatever is baked into the image.